apps/src/templates/sectionsRefresh/AdvancedSettingToggles.jsx

```diff
@@ -76,11 +78,15 @@ export default function AdvancedSettingToggles({
           />
         </div>
       )}
-      {assignedUnitLessonExtrasAvailable && (
+      {hasLessonExtras && (
         <div className={style.toolTipContainer}>
           <ToggleSwitch
             id={'uitest-lesson-extras-toggle'}
-            isToggledOn={section.lessonExtras}
+            isToggledOn={
```

Comment:
is lessonExtras ever undefined? I am wondering if an || would work here

Reply:
Additionally, you can use the '?' instead of checking if something is equal to undefined!